<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ice_box</class>
 <widget class="QWidget" name="ice_box">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>429</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Icebox</string>
  </property>
  <widget class="QLabel" name="ice_box_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>    Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="ice_box_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>     -2</string>
   </property>
  </widget>
  <widget class="QLabel" name="ice_box_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cooling chamber</string>
   </property>
  </widget>
  <widget class="QLabel" name="ice_box_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>            Freezer</string>
   </property>
  </widget>
  <widget class="QLabel" name="ice_box_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>      -8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp_plan_high">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>      --</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp_plan_down">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>      --</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_temp_plan_plus">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_temp_plan_minus">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
